Polyhedron/demo/Polyhedron/MainWindow.ui

```diff
@@ -676,6 +677,14 @@
     <string>Set Maximum  Text Items Displayed :</string>
    </property>
   </action>
+  <action name="actionSwitchProjection">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Orthographic/Perspective Projection</string>
```

Comment:
If the action is checkable, its name shoudl be the name of the state once check: "Use orthographic projection instead".